### ISSUE TITLE
[BUG] Exiting back to main menu completely breaks mod loader pro.

### ISSUE THREAD

**Issue by BishopModdist** [MSCLoaderPro/MSCModLoaderPro#25]
**Describe the bug**
After saving or exiting to main menu, the mod loader breaks - version is not shown, nexus mods login reads "username", console reads "gfdrgdfgfgdf", controls unresponsive and other. Screenshot shows more.

**To Reproduce**
Play the game with mods enabled, either save or exit to menu. Voila.

**Expected behavior**
I expected it to just send me back to the menu without me having to restart the game.

**Screenshots**
![what](https://user-images.githubusercontent.com/68397200/122553943-9b77d580-d038-11eb-96fb-ed1c236eba64.png)

**output_log.txt**
Please provide the "output_log.txt" file form My Summer Car folder.
[output_log.txt](https://github.com/MSCLoaderPro/MSCModLoaderPro/files/6676718/output_log.txt)

**Comment by Tresquel** [comment]
It seems like your mod "Mono.Cecil.Rocks.dll" is making the loader go crazy, try to rename the mod to something like "Mono-Cecil-Rocks.dll" and it should work

**Comment by Fredrik95** [comment]
Yes, like Tresquel said, the Mono.Cecil.Rocks.dll is causing issues..

I don't think that's a mod at all, try removing it from the mods folder.

If that doesn't work, report back!
